test/cmd/get.sh

```diff
@@ -22,24 +22,18 @@ os::cmd::expect_success_and_not_text "_output/oshinko get abc -o json --nopods"
 # get all
 os::cmd::expect_success_and_text "_output/oshinko get" "abc"
 os::cmd::expect_success_and_text "_output/oshinko get" "def"
- 
+
 # check for columns
-os::cmd::expect_success '_output/oshinko-cli get abc | grep -e "^abc\s*[012]\s*spark://abc:7077\s*http://abc-ui:8080\s*abc-ui-route.*\s*Running\s*<shared>$"'
```

Comment:
Here too, we should probably include column checks for the -d case since it's a feature you can turn on and off